library/twirp-go/v8.1.2/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/bufbuild/plugins/contrib/twitchtv
+
+go 1.19
+
+require (
+	github.com/twitchtv/twirp v8.1.2+incompatible
+	google.golang.org/protobuf v1.28.1
+)
```

Comment:
Followed instructions here: https://twitchtv.github.io/twirp/docs/install.html#define-toolsgo-for-versioning-in-gomod

Until the upstream repo adds support for Go modules, this is how we'll guarantee building with consistent set of module dependencies.